Polyhedron_IO/include/CGAL/IO/STL_writer.h

```diff
@@ -85,11 +85,12 @@ write_STL(const TriangleMesh& tm, std::ostream& out)
 
       Vector_3 n = collinear(p,q,r) ? Vector_3(1,0,0):
                                       unit_normal(p,q,r);
-      out << "facet normal " << n << "\nouter loop\n";
-      out << "vertex " << p << "\n";
-      out << "vertex " << q << "\n";
-      out << "vertex " << r << "\n";
-      out << "endloop\nendfacet\n";
+      out << std::scientific << std::setprecision(6);
```

Comment:
I disagree with `std::setprecision(6)`. Would you accept to change it into `std::setprecision(17)`, so that the double precision (with a 53 bits mantissa) is preserved during the input/ouputs?

Reply:
Agree, wasn't sure if there was a character limit on paraview. Adjusting to 

```suggestion
      out << std::scientific << std::setprecision(17);
```

should work.